recipes/cminpack/all/conanfile.py

```diff
@@ -0,0 +1,110 @@
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["BUILD_EXAMPLES"] = "OFF"
+        tc.variables["CMINPACK_LIB_INSTALL_DIR"] = "lib"
+        tc.cache_variables["CMAKE_POLICY_DEFAULT_CMP0077"] = "NEW"
+        tc.generate()
+
+    def layout(self):
+        cmake_layout(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            try:
+                del self.options.fPIC
+            except Exception:
+                pass
+
+        # cminpack is a c library
+        try:
+            del self.settings.compiler.libcxx
+        except Exception:
+            pass
+        try:
+            del self.settings.compiler.cppstd
+        except Exception:
+            pass
+
+    def source(self):
+        files.get(self, **self.conan_data["sources"][self.version],
+                  strip_root=True, destination=self.source_folder)
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        cmake = CMake(self)
+        cmake.install()
+
+        files.rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
+        files.copy(self, "CopyrightMINPACK.txt", dst=os.path.join(self.package_folder, "licenses"), src=self.source_folder)
+        files.rmdir(self, os.path.join(self.package_folder, "share")) # contains cmake config files
+
+    def _library_postfix(self):
+        postfix = ""
+        if not self.options.shared:
+            postfix += "_s" # for static
+        if self.settings.build_type == "Debug":
+            postfix += "_d"
+
+        return postfix
+
+    def package_info(self):
+        minpack_include_dir = os.path.join("include", "cminpack-1")
+        
+        # the double precision version
+        self.cpp_info.components['cminpackd'].libs = ['cminpack' + self._library_postfix()]
+        self.cpp_info.components['cminpackd'].includedirs.append(minpack_include_dir)
+        self.cpp_info.components["cminpackd"].set_property("cmake_target_name", "cminpack::cminpack")
+        self.cpp_info.components["cminpackd"].names["cmake_find_package"] = "cminpack"
+        self.cpp_info.components["cminpackd"].names["cmake_find_package_multi"] = "cminpack"
+        
+        # the single precision version
+        self.cpp_info.components['cminpacks'].libs = ['cminpacks' + self._library_postfix()]
+        self.cpp_info.components['cminpacks'].includedirs.append(minpack_include_dir)
+        self.cpp_info.components['cminpacks'].defines.append("__cminpack_float__")
+
+        if self.settings.os != "Windows":
+            self.cpp_info.components['cminpackd'].system_libs.append("m")
+            self.cpp_info.components['cminpacks'].system_libs.append("m")
```

Comment:
I thought these were duplicates are first glance... 🙈 

Any objections to being a little more verbose?

```suggestion
            self.cpp_info.components['cminpack_double'].system_libs.append("m")
            self.cpp_info.components['cminpack_single'].system_libs.append("m")
```